pyrasterframes/src/main/python/docs/aggregation.pymd

```diff
@@ -10,13 +10,65 @@ import os
 spark = create_rf_spark_session()
 ```
 
-## Cell Counts & Tile Mean
+There are 3 types of aggregate functions: tile aggregate, DataFrame aggregate, and element-wise local aggregate. In the @ref:[`tile aggregate functions`](reference.md#tile-statistics), we are computing a statistical summary per row of a `tile` column in a DataFrame. In the @ref:[`DataFrame aggregate functions`](reference.md#aggregate-tile-statistics), we are computing statistical summaries over all of the cell values *and* across all of the rows in the DataFrame or group. In the  @ref:[`element-wise local aggregate functions`](reference.md#tile-local-aggregate-statistics), we are computing the element-wise statistical summary across a DataFrame or group of tiles.
+
+## Tile Mean
+
+We can illustrate this difference in computing tile mean. In the first code block we are using @ref:[`rf_tile_mean`](reference.md#rf-tile-mean) to computer the tile aggregate mean of cells in each row of column `tile`. The mean of each tile is computed separately, so the first mean is 1.0 and the second mean is 3.0.
```

Comment:
Some spelling and such 

May be clarifying here to point out that the number of rows in the dataframe will be the same before and after.

Could be worth `show`ing the result of the spark.sql (it should pretty print nicely) as well to reinforce what's happening.

```suggestion
We can illustrate these differences in computing tile mean. In the first code block we are using @ref:[`rf_tile_mean`](reference.md#rf-tile-mean) to compute the tile aggregate mean of cells in each row of column `tile`. The mean of each tile is computed separately, so the first mean is 1.0 and the second mean is 3.0.
```